app/models/grda_warehouse/hud/project.rb

```diff
@@ -858,6 +858,13 @@ def participating_in_ce_on?(date)
       end.present?
     end
 
+    # NOTE: preload funders before calling this
+    def pay_for_success?
+      return false unless project_type == 7
```

Comment:
Something like this may help future-proof against for HUD changes. 

```suggestion
      return false unless HudUtility2024.performance_reporting[:other].include?(project_type)
```